src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int SCALE_FACTOR = 2;
+    private Node<K, V>[] table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+    private int capacity = DEFAULT_CAPACITY;
+    private int threshold = (int) (DEFAULT_LOAD_FACTOR * DEFAULT_CAPACITY);
```

Comment:
threshold is a useless field, please remove it
we read its value only once, not worth it
capacity can be obtained at any moment by calling .length on table
init table in constructor, as prev. mentioned